From eba63277b467c7a81962e51096c776dc8efb1b81 Mon Sep 17 00:00:00 2001
From: Tamizh Chelvam Raja <quic_tamizhr@quicinc.com>
Date: Fri, 2 Sep 2022 19:47:19 +0530
Subject: [PATCH 1/4] cfg80211: Fix mcs divisor value

mcs divisor for mcs6 is wrongly calculated, Fix it by proper value

previous mcs_divisor value = (11769/6144) = 1.915527

fixed mcs_divisor value = (11377/6144) = 1.851725

With the previous mcs_divisor value, bitrate for mcs6 calculated
wrongly.

Change-Id: I822bd2e690a6b00eb01c77060327f7e40a55f37b
Signed-off-by: Tamizh Chelvam Raja <quic_tamizhr@quicinc.com>
---
 net/wireless/util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/wireless/util.c b/net/wireless/util.c
index 6f6d806..ffdb1a7 100644
--- a/net/wireless/util.c
+++ b/net/wireless/util.c
@@ -1377,7 +1377,7 @@ static u32 cfg80211_calculate_bitrate_he(struct rate_info *rate)
 		 25599, /*  4.166666... */
 		 17067, /*  2.777777... */
 		 12801, /*  2.083333... */
-		 11769, /*  1.851851... */
+		 11377, /*  1.851725... */
 		 10239, /*  1.666666... */
 		  8532, /*  1.388888... */
 		  7680, /*  1.250000... */
-- 
2.17.1

